tests/component-tests/uploader/js/HTML5UploaderSupportTests.js

```diff
@@ -201,9 +201,10 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
 
             jqUnit.module("Uploader Basic Tests");
 
-            var file1 = fluid.tests.uploader.html5.mockFile("file1", "emptyfile.zip", "application/zip", 0),
+            var file1 = fluid.tests.uploader.html5.mockFile("file1", "emptyfile.zip", "application/zip", 1024),
```

Comment:
It seems that this first one was specifically an empty file, as the name suggest. May instead make another new file for the 1KB size.